Lib/test/test_embed.py

```diff
@@ -1768,6 +1768,8 @@ def test_init_set_config(self):
         self.check_all_configs("test_init_set_config", config,
                                api=API_ISOLATED)
 
+    @unittest.skipIf("-D_Py_JIT" in (sysconfig.get_config_var('PY_CORE_CFLAGS') or ''),
```

Comment:
This fails on windows for some reason

Reply:
I think with my fix we can actually run the test so I am switching to that

Reply:
Hmm I'm abit stumped. No clue why your fix isn't working :(.

Reply:
I pushed a fix that should do the trick